packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseTrendingProjects.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useGetQueryResultBundleWithAsyncStatus } from '../../synapse-queries'
+import { BUNDLE_MASK_QUERY_RESULTS } from '../../utils/SynapseConstants'
+import {
+  SynapseTrendingProjectItem,
+  trendingProjectsGridTemplateColumns,
+} from './SynapseTrendingProjectsItem'
+import { Box, useTheme, useMediaQuery } from '@mui/material'
+import { Typography } from '@mui/material'
+
+export type SynapseTrendingProjectsProps = {
+  past30DaysDownloadMetricsTable: string
+}
+
+export const SynapseTrendingProjects: React.FunctionComponent<
+  SynapseTrendingProjectsProps
+> = ({ past30DaysDownloadMetricsTable }) => {
+  const { data: past30DaysDownloadData } =
+    useGetQueryResultBundleWithAsyncStatus({
+      entityId: past30DaysDownloadMetricsTable,
+      query: {
+        sql: `SELECT * FROM ${past30DaysDownloadMetricsTable}`,
+        limit: 10,
+        sort: [
+          { column: 'last_updated', direction: 'DESC' }, // First sort by export date desc (only the most recent export)
+          { column: 'N_UNIQUE_USERS', direction: 'DESC' }, // TODO: Is this the correct secondary sort?
```

Comment:
I think so, but I don't think that's my decision to make 🙂

Reply:
Updated